utils/meta-intel-sdo/recipes-connectivity/c-code-sdk/c-code-sdk.bb

```diff
@@ -1,51 +0,0 @@
-DESCRIPTION="Recipe for sdo (c-code-sdk) on mbed-linux"
-TOOLCHAIN = "POKY-GLIBC"
-LICENSE = "CLOSED"
-S = "${WORKDIR}"
-
-DEPENDS += "mbedtls"
-
-INSANE_SKIP_${PN} = "dev-elf"
-INSANE_SKIP_${PN} = "ldflags"
-INSANE_SKIP_${PN}-dev = "ldflags"  
-PACKAGES = "${PN}"
-INHIBIT_PACKAGE_DEBUG_SPLIT = "1"
-INHIBIT_PACKAGE_STRIP = "1"
-INHIBIT_SYSROOT_STRIP = "1"
-FILES_${PN} += "/opt \
-                /opt/arm \
-                /opt/arm/linux-client" 
-
-
-FILESEXTRAPATHS_prepend := "${THISDIR}:"
-SRC_URI = "file://sdo.tar.gz"
-
-
-do_configure(){
-}
-
-do_compile(){
-CUR_DIR=$(pwd)
-cd "${WORKDIR}/sdo"
-export CFLAGS="-D_IPP_v50_ -D_IPP_DEBUG -D_DISABLE_ALG_MD5_ -mfloat-abi=hard -D_DISABLE_ALG_SM3_ -Wstrict-aliasing -g -D_FORTIFY_SOURCE=2 HTTPPROXY=false"
```

Comment:
this is needed for ARM build. how is this handled in the new .bb files??

Reply:
by cmake.

Reply:
currently its not in cmake.